src/main/scala/io/sdkman/changelogs/ScalaMigrations.scala

```diff
@@ -260,4 +260,95 @@ class ScalaMigrations {
         "3.0.0-RC1/scala3-3.0.0-RC1.zip"
     ).validate()
       .insert()
+
+  @ChangeSet(
+    order = "020",
+    id = "020_add_scala_2.10.1",
+    author = "eliso"
+  )
+  def migration020(implicit db: MongoDatabase): Unit =
+    Version(
+      candidate = "scala",
+      version = "2.10.1",
+      url = "https://downloads.lightbend.com/scala/2.10.1/scala-2.10.1.zip"
+    ).validate()
+      .insert()
```

Comment:
How about trying some like this instead?

```suggestion
List("2.10.1", "2.10.2", "2.10.3", .....).map { version =>
  Version(
      candidate = "scala",
      version = version,
      url = "https://downloads.lightbend.com/scala/${version}/scala-${version}.zip"
  )
}.validate().insert()
```

Reply:
Thanks for you comment!

I've actually generated those methods by iterating on the versions - as you pointed out - and then pasted them in the class.
I totally agree with this. I will change that.

Reply:
Changes pushed. Let me know!